h2integrate/controllers/openloop_controllers.py

```diff
@@ -0,0 +1,130 @@
+import openmdao.api as om
+from attrs import field, define
+from numpy import ones
+
+from h2integrate.core.utilities import BaseConfig, merge_shared_inputs
+
+
+class ControllerBaseClass(om.ExplicitComponent):
```

Comment:
Should this bee ControllerBaseClass or OpenLoopControllerBaseClass? Are we going to use this for all controllers?

Reply:
The edits made by @bayc made this more generic so it could potentially be used as a base class for closed-loop controllers as well as open-loop controllers. I think that approach makes sense and it can evolve as needed.